urx/ursecmon.py

```diff
@@ -50,6 +50,7 @@ class ParserUtils(object):
     def __init__(self):
         self.logger = logging.getLogger("ursecmon")
         self.is_v30 = False
+        self.is_v32 = False
```

Comment:
this is a bit of a hack, we should have one variable that is set to the version, not a variable per version. Do you feel able to fix this?